### DISCUSSION TITLE
New Ternary Formatting: add experimentalTernaries from pritter

### DISCUSSION THREAD

**Discussion by meded90** [biomejs/biome#3359] [General]
We maintain a large project that relies heavily on the experimentalTernaries feature from Prettier, and our team is accustomed to this formatting style. We request the implementation of a similar feature in BiomeJS for the following reasons:

- Consistency: Maintaining the same formatting style across our project.
- Readability: Improved readability for deeply nested ternary operations.
- Adoption: Attracting users familiar with Prettier’s formatting.

This feature will greatly enhance our workflow and code quality. We are eager to see this functionality in BiomeJS.

https://github.com/prettier/prettier/pull/13183

**Comment by ematipico** [comment]
I'm not against the proposal, but I'm against developing something that is experimental.

Experimental means:
- it can be dropped 
- it can change scope anytime 
- it's buggy 

Given that, I don't want the team to take on a project so unstable, and waste so many resources.

**Reply by RedHatter** [reply]
Would you be amiable to a PR that added support?

**Reply by ematipico** [reply]
I can't prevent you from contributing, however the maintenance of the feature would fall on us, unless you decide to stick around to fix bugs. This means that it's healthier for the project to wait for how the experiment goes
